experimental/AlgebraicShifting/src/AlgebraicShifting.jl

```diff
@@ -0,0 +1,119 @@
+include("UniformHypergraph.jl")
+include("PartialShift.jl")
+include("PartialShiftGraph.jl")
+
+export UniformHypergraph
+
+export compound_matrix
+export contracted_partial_shift_graph
+export exterior_shift
+export face_size
+export generic_unipotent_matrix
+export partial_shift_graph
+export partial_shift_graph_vertices
+export rothe_matrix
+export uniform_hypergraph
+
+function independent_columns(A::MatElem)
+  col_indices = Int[]
+  row_index = 1
+  row_bound = nrows(A)
+  for col_index in 1:ncols(A)
+    if !is_zero(A[row_index, col_index])
+      push!(col_indices, col_index)
+      row_index += 1
+    end
+    row_index > row_bound && break
+  end
+  return col_indices
+end
+
+
+"""
+    symmetric_shift(F::Field, K::SimplicialComplex)
+
+Returns the symmetric shift of `K`
+
+see survey on algebraic shifting Gil Kalai (properly cite at some point)
```

Comment:
Resolve this TODO? But also OK to defer it to a follow-up PR, but then I'd recommend to at least make it easily grepable, e.g.
```suggestion
See survey on algebraic shifting Gil Kalai (TODO: properly cite at some point)
```